home/templates/accounts/sign-in-client.html

```diff
@@ -0,0 +1,80 @@
+                                <img src="{% static 'assets/img/pages/login/favicon.ico' %}" alt="Logo" class="mb-3">
+                                <h1 class="mb-0 h3">Sign in to our platform</h1>
+                            </div>
+
+                            <form method="post" class="mt-4">
+                                {% csrf_token %}
+                                <!-- Form -->
+                                {% if form.non_field_errors %}
+                                    {% for error in form.non_field_errors %}
+                                        <span class="text-danger"> {{ error }} </span>
+                                    {% endfor %}
+                                {% endif %}
+                                {% for field in form %}
+                                <div class="form-group mb-4">
+                                    <label for="email">{{ field.label }}</label>
+                                    <div class="input-group">
+                                        <span class="input-group-text" id="basic-addon1">
+                                            <span class="fas {% if field.name == 'username' %} fa-user {% else %} fa-unlock-alt {% endif %}"></span>
+                                        </span>
+                                        {{ field|add_input_validation }}
+                                    </div>  
+                                </div>
+                                <span class="text-danger"> {{ field.errors }} </span>
+                                {% endfor %}
+                                <!-- End of Form -->
+                                <div class="d-flex justify-content-between align-items-center mb-4">
+                                    <div class="form-check mb-0">
+                                        <input class="form-check-input" type="checkbox" value="" id="remember">
+                                        <label class="form-check-label mb-0" for="remember">
+                                            Remember me
+                                        </label>
+                                    </div>
+                                    <div><a href="{% url 'password_reset' %}" class="small text-right lost-password">Lost password?</a></div>
+                                </div>
+                                <div class="d-grid">
+                                    <button type="submit" class="btn btn-warning btn-primary">Sign in</button>
+                                </div>
+                            </form>
+
+                            <div class="d-flex justify-content-center align-items-center mt-4">
+                                <span class="fw-normal">
+                                    Not registered?
+                                    <a href="{% url 'register' %}" class="fw-bold text-underline create-account">Create account</a>
```

Comment:
Please update the sign-in page to reference "registerClient," using the format from your sign-up and url.

Reply:
According to hiichloee's PR (PR 214), a personal email was provided, but the sign-up process requires a Deakin email. Please discuss this with hiichloee and make the necessary adjustments.

Reply:
@Armaanooo fix this change too